core/config/docs/core.toml

```diff
@@ -470,6 +470,12 @@ MaxEncryptedSecretsSize = '26.40kb' # Default
 # MaxConfigSize is the maximum size of a config that can be fetched from the given config url.
 MaxConfigSize = '50.00kb' # Default
 
+[Capabilities.WorkflowRegistry.Limits]
```

Comment:
I still think we should move this to something that doesn't imply it's a limit that only applies to the WorkflowRegistry

eg. [Workflows.Limits]